src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+import java.util.Arrays;
+import java.util.Scanner;
 public class Duke {
+    private static Scanner in = new Scanner(System.in);
+    private static int taskNumber = 0;
+    private static Task[] t = new Task[100];
+    private static String line = "*********************************************";
+
+    private static void helloMessage() {
+        System.out.println(line);
+        System.out.println("Hello! I'm Julia");
+        System.out.println("What can I do for you?");
+        System.out.println();
+        System.out.println(line);
+    }
+    private static void byeMessage() {
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println();
+        System.out.println(line);
+        System.exit(0);
+    }
+
+    public static void printList(Task[]t){
+        int count=1;
+        System.out.println("Here are the tasks in your list: ");
+        for(int i=0; i<taskNumber;i++) {
```

Comment:
This is the correct if-conditional layout with the 'break' in a separate line, well done!